### DISCUSSION TITLE
Expiring Tokens

### DISCUSSION THREAD

**Discussion by estone4009** [felixwttr/Dropbox_auto_deletion#2] [Q&A]
Has anyone found a way to use this currently with the tokens that now expire?  This works but I think the tokens expire every 4 hours so it is shor tlived.

**Comment by felixwttr** [comment]
Hi @estone4009 ,
currently I am busy studying. I will have a look into this when I have time for it.
It's on my ToDo ;)
Thank you for using my script. :)
Regards
Felix
